lib/routes/bins/update.js

```diff
@@ -38,20 +41,30 @@ module.exports = (client) => (req, res, next) => {
 
 	mock.content.size = 0;
 	// express is sensitive to mismatched content-length header
+	mock.headers ||= [];
 	mock.headers = mock.headers.filter(
 		(header) => header.name.toLowerCase() !== "content-length",
 	);
 
 	validate
 		.response(mock)
 		.then(function () {
+			// method-scoped mock
 			client.set(
 				`bin:${compoundId}`,
 				JSON.stringify(mock),
 				"EX",
 				process.env.MOCKBIN_REDIS_EXPIRE_SECONDS || "1000000000",
 			);
 
+			// also set method-less mock (to preserve compatibility with older versions of Insomnia)
+			client.set(
+				`bin:${oldCompoundId}`,
+				JSON.stringify(mock),
+				"EX",
+				process.env.MOCKBIN_REDIS_EXPIRE_SECONDS || "1000000000",
+			);
+
```

Comment:
have you considered the approach taken in the PR which preceded this one? In brief a missing method in the key equals GET method